Vimeo-Native/NugetReferencer/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="5.0.5" targetFramework="net40" />
-  <package id="VimeoAPI" version="1.0.1.0" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
+  <package id="VimeoAPI" version="1.0.1.0" targetFramework="net4" />
```

Comment:
It seems like that must have slipped through the cracks of our dependency list generation. We only had the native Vimeo packages listed. I'll make a new card for the .NET side and get that built first.